src/wrapper.rs

```diff
@@ -482,6 +482,52 @@ impl DxcLibrary {
     }
 }
 
+#[derive(Debug)]
+pub struct DxilContainerReflection {
+    inner: ComPtr<IDxcContainerReflection>,
+}
+
+impl DxilContainerReflection {
+    pub fn new(inner: ComPtr<IDxcContainerReflection>) -> Self {
+        Self { inner }
+    }
+
+    pub fn load(&self, p_dxc_blob: &IDxcBlob) {
+        unsafe {
+            self.inner.load(p_dxc_blob);
+        }
+    }
+
+    pub fn find_first_part_kind(&self) -> u32 {
+        let mut shader_idx = 0u32;
+        unsafe {
+            self.inner.find_first_part_kind(
+                Self::four_cc('D' as u32, 'X' as u32, 'I' as u32, 'L' as u32),
+                &mut shader_idx,
+            );
+        }
+        shader_idx
+    }
+
+    fn four_cc(ch0: u32, ch1: u32, ch2: u32, ch3: u32) -> u32 {
```

Comment:
You may write [`fourcc`](https://www.fourcc.org/) as one word; reads easier and it's the name of the thing.

Reply:
Shouldn't these be `u8` btw?